staff/francisco-fernandez/cv/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Curriculum vitae de Oscar Medina</title>
+</head>
+<body>
+
+    <section id="headercv">
```

Comment:
ids should be avoided, do not use ids for no explicit reason. they are not part of html semantics